core/src/main/java/com/linecorp/armeria/common/DeferredHttpResponse.java

```diff
@@ -69,4 +80,43 @@ public EventExecutor defaultSubscriberExecutor() {
         }
         return super.defaultSubscriberExecutor();
     }
+
+    @Override
+    public void subscribe(Subscriber<? super HttpObject> subscriber, EventExecutor executor,
+                          SubscriptionOption... options) {
+        super.subscribe(new Subscriber<HttpObject>() {
```

Comment:
We already wrap the `Subscriber` given by a user in `DeferredStreamMessage`. Would it be possible to consolidate this into `DeferredStreamMessage` somehow so we don't have to wrap the `Subscriber` twice?